src/main/java/org/spongepowered/api/service/event/EventManager.java

```diff
@@ -34,13 +35,23 @@
     /**
      * Registers an object to receive {@link Event}s.
      *
-     * @param plugin A plugin instance
+     * @param plugin The plugin instance
      * @param obj The object
      * @throws IllegalArgumentException Thrown if {@code plugin} is not a plugin instance
      */
     void register(Object plugin, Object obj);
```

Comment:
> We aren't renaming this method

(Why not?) Are you adding some clarifications to the javadocs?

> We also aren't returning the handlers as it would have to be immutable and that would really do no good for you.

It would be quite useful to be able to realign the priority of the event handlers to better match another plugin's handler's priority. Example execute the Login plugin's handler after the daily join gift handler.